CHANGELOG

```diff
@@ -6,28 +6,34 @@
                 _/    _/  _/        _/  _/    _/      _/_/
                  _/_/_/  _/_/_/_/    _/      _/  _/_/_/
 
-                              http://glvis.org
+                             https://glvis.org
 
 
-Version 4.1.1 (development)
-===========================
-
-- Added the option to cut a portion of the interiors of 3D faces to expose more
-  of the mesh. Useful as an alternative to transparency. See keys Ctrl+F3/F4.
+Version 4.2 released on May 20, 2022
```

Comment:
Update date?

Reply:
Done in https://github.com/GLVis/glvis/pull/227/commits/17e2a3cd6ea843ed9d7bd8213c63ce16d942345d